pkgs/development/python-modules/pySMART_smartx/default.nix

```
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pySMART.smartx";
```

```suggestion
  pname = "pysmart-smartx";
```

See https://nixos.org/manual/nixpkgs/unstable/#contributing